standalone/src/models/associate-role/associate-role-draft/presets/empty.ts

```diff
@@ -1,11 +1,28 @@
-import type { TAssociateRoleDraftBuilder } from '../../types';
-import AssociateRoleDraft from '../builder';
+import { TBuilder } from '@/core';
+import type {
+  TAssociateRoleDraft,
+  TAssociateRoleDraftGraphql,
+  TAssociateRoleDraftRest,
+} from '../../types';
+import {
+  RestModelBuilder,
+  GraphqlModelBuilder,
+  CompatModelBuilder,
+} from '../builders';
 
-const empty = (): TAssociateRoleDraftBuilder =>
-  AssociateRoleDraft()
-    .name(undefined)
-    .buyerAssignable(undefined)
-    .permissions(undefined)
-    .custom(undefined);
+const populatePreset = <
+  TModel extends TAssociateRoleDraftGraphql | TAssociateRoleDraftRest,
+>(
+  builder: TBuilder<TModel>
+) => {
+  return builder.name(null).permissions([]).custom(null);
```

Comment:
I realize that you are updating this preset to return values that are not optional in the `Draft` types, but should `buyerAssignable` be set to `null` for backwards compat, or is part of this process updating test files that consume this preset if/when necessary once this package is released?

Reply:
That's a good question/

I thought it doesn't make sense to set to `null` required values so I updated the preset.

I also tested this changes in my local MC frontend repo and nothing failed.